src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/Browsing/SearchPage.xaml

```diff
@@ -0,0 +1,54 @@
+﻿<Page
+    x:Class="OfflineWorkflowSample.Views.SearchPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <SplitView DataContext="{Binding Source={StaticResource MainViewModel},Path=PortalViewModel.SearchViewModel}"
+               IsPaneOpen="True"
+               DisplayMode="Inline"
+               PaneBackground="Transparent">
+        <SplitView.Pane>
+            <ScrollViewer>
+                <StackPanel Padding="5,0,20,0">
+                    <AutoSuggestBox Text="{Binding SearchQueryText,Mode=TwoWay}"
+                                    PlaceholderText="Search Portal" />
+                    <CheckBox Content="Include public results"
+                              IsChecked="{Binding IncludePublicResults,Mode=TwoWay}" />
+                    <Grid ColumnSpacing="5">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
+                        <Button Content="Prev"
+                                Grid.Column="0"
+                                HorizontalAlignment="Stretch"
+                                Command="{Binding GoBackCommand}" />
+                        <TextBlock Text="{Binding Page,Mode=TwoWay}"
+                                   VerticalAlignment="Center"
+                                   HorizontalAlignment="Center"
+                                   Grid.Column="1" />
+                        <Button Content="Next"
+                                Grid.Column="2"
+                                HorizontalAlignment="Stretch"
+                                Command="{Binding GoForwardCommand}" />
+                    </Grid>
+                    <TextBlock>
+                        <Run>Total results: </Run>
+                        <Run Text="{Binding TotalResults}" />
+                    </TextBlock>
+                </StackPanel>
+            </ScrollViewer>
+        </SplitView.Pane>
+        <SplitView.Content>
+            <GridView ItemsSource="{Binding SearchResults}"
+                      ItemTemplate="{StaticResource PortalItemTemplate}"
+                      SelectedItem="{Binding Source={StaticResource MainViewModel},Path=SelectedItem,Mode=TwoWay}"
+                      Margin="5,0,5,0" />
+        </SplitView.Content>
+    </SplitView>
```

Comment:
Use x:Bind instead of Binding

Reply:
I tried switching all of these to x:Bind, but UWP doesn't seem to do a good job handling multiple two-way bindings to the same item. When setting selected item, it will change the property, then try to update the property for every other binding to that same property, which are still null. 

Binding doesn't have the same problem.